src/directive/user.ts

```diff
@@ -26,6 +28,23 @@ export const deleteUser = async (
 ) => {
   try {
     await con.transaction(async (entityManager): Promise<void> => {
+      const user = await entityManager.getRepository(User).findOne({
+        select: ['subscriptionFlags'],
+        where: { id: userId },
+      });
+      if (user?.subscriptionFlags?.subscriptionId) {
+        await cancelSubscription({
+          subscriptionId: user.subscriptionFlags.subscriptionId,
+        });
+        logger.info(
+          {
+            type: 'paddle',
+            userId,
+            subscriptionId: user.subscriptionFlags.subscriptionId,
+          },
+          'Subscription cancelled user deletion',
+        );
+      }
```

Comment:
Ideally it should be before the transaction. I don't want to hold the transaction hostage while we wait for paddle